src/elements/core/styles/mixins/link.scss

```diff
@@ -6,11 +6,7 @@
   --sbb-link-color-normal: var(--sbb-color-granite);
   --sbb-link-color-hover: var(--sbb-color-red125);
   --sbb-link-color-active: var(--sbb-color-red150);
-  --sbb-link-text-decoration: none;
-
-  @include a11y.if-forced-colors {
-    --sbb-link-text-decoration: underline;
-  }
+  --sbb-link-text-decoration: var(--sbb-link-text-decoration-none, underline);
```

Comment:
I still don't get why we need this extra variable?

Reply:
I discussed this later with Lukas and the problem is that if I override the `--sbb-link-text-decoration` from the footer it doesn't work, so I need the second var with the fallback approach.

Reply:
Shouldn't we remove "--sbb-link-text-decoration" here and apply it as a fallback where it is currently used?

Reply:
(to avoid duplicating var)